docs/resources/integration_jira.md

```diff
@@ -0,0 +1,58 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "mondoo_integration_jira Resource - terraform-provider-mondoo"
+subcategory: ""
+description: |-
+  Integrate the Ticketing System Jira with Mondoo to automatically create and close issues based on Mondoo findings.
+---
+
+# mondoo_integration_jira (Resource)
+
+Integrate the Ticketing System Jira with Mondoo to automatically create and close issues based on Mondoo findings.
+
+## Example Usage
+
+```terraform
+variable "jira_token" {
+  description = "The Jira API Token"
+  type        = string
+  sensitive   = true
+}
+
+provider "mondoo" {
+  space = "hungry-poet-123456"
+}
+
+# Setup the Jira integration
+resource "mondoo_integration_jira" "jira_integration" {
+  name  = "My Jira Integration"
+  host  = "https://your-instance.atlassian.net"
+  email = "jira.owner@email.com"
+  # default_project = "MONDOO"
+  api_token = var.jira_token
```

Comment:
Token needs to become a credentials block, to align with the other integrations:

```
 credentials = {
    api_token = var.jira_token
  }
  ```

Reply:
Same ☝🏽